topostats/tracing/tracingfuncs.py

```diff
@@ -531,3 +531,26 @@ def coord_dist(coords: npt.NDArray, px_2_nm: float = 1) -> npt.NDArray:
             dist += 1
         dist_list.append(dist)
     return np.asarray(dist_list) * px_2_nm
+
+
+def get_two_combinations(list_of_values) -> list:
+    """
+    Obtain all paired combinations of values in the list.
+
+    Example: [1,2] -> [[1,2]], [1,2,3] -> [[1,2],[1,3],[2,3]]
```

Comment:
Perhaps split the two examples onto separate lines, we can also use in-line examples as shown in [Numpydoc Style](https://numpydoc.readthedocs.io/en/latest/format.html#examples).

Suggest moving this to after the `Return` section as...

```suggestion
    Examples
    --------
    >>> get_two_combinations([1,2])
    array([[1,2]])
    >>> get_two_combinations([1,2,3])
    array([[1,2],[1,3],[2,3]])
```

Examples can also be used as [doctests](https://docs.pytest.org/en/8.3.x/how-to/doctest.html) so I'm keen to have more of these throughout the code base.

Reply:
get_two_combs removed for itertools.combinations :)